### PR TITLE
Apply theme to web client

### PR DESCRIPTION
**Pull request by Enderedne**
### Describe the background of your pull request

Apply the theme Kalina defined in https://bareynol.github.io/mui-theme-creator/. The exact parameters will be available in the ux-guidelines doc after the review is complete.

### Governance 

- [ ] Documentation is added
- [ ] Test cases are added or updated

By submitting this pull request I confirm that:
- I've read the contributing document https://github.com/alkem-io/.github/blob/master/CONTRIBUTING.md.
- I've read and understand the Code of Conduct https://github.com/alkem-io/.github/blob/master/CODE_OF_CONDUCT.md.
- I understand that any contributions or suggestions I made may make it into the actual code. I've read the License 
     https://github.com/alkem-io/Coordination/blob/master/LICENSE.
- I understand that submitting the pull request requires agreeing to and signing the contributor license agreement.

### CODE REVIEW

File: src/pages/Discussions/DiscussionPage.tsx [hunk 1]
```diff
@@ -73,7 +73,7 @@ export const DiscussionPage: FC<DiscussionPageProps> = ({ paths }) => {
   };
 
   return (
-    <ThemeProviderV2>
+    <ThemeProvider>
```
Comment (atstoyanov) [inline src/pages/Discussions/DiscussionPage.tsx:76]: Not needed here if there is just one theme.

File: src/pages/Discussions/NewDiscussionPage.tsx [hunk 2]
```diff
@@ -22,13 +22,13 @@ const NewDiscussionPage: FC<NewDiscussionPageProps> = ({ paths }) => {
   const title = `${communityName} - Initiate Discussion`;
 
   return (
-    <ThemeProviderV2>
+    <ThemeProvider>
```
Comment (atstoyanov) [inline src/pages/Discussions/NewDiscussionPage.tsx:25]: Not needed here if there is just one theme.

File: src/pages/Organization/OrganizationPage.tsx [hunk 3]
```diff
@@ -11,13 +11,13 @@ export const OrganizationPage: FC<OrganizationPageProps> = ({ paths }) => {
   useUpdateNavigation({ currentPaths: paths });
 
   return (
-    <ThemeProviderV2>
+    <ThemeProvider>
```
Comment (atstoyanov) [inline src/pages/Organization/OrganizationPage.tsx:14]: Not needed here if there is just one theme.

File: src/pages/User/UserProfilePage.tsx [hunk 4]
```diff
@@ -29,9 +29,9 @@ export const UserProfilePage: FC<UserProfileProps> = () => {
   };
 
   return (
-    <ThemeProviderV2>
+    <ThemeProvider>
```
Comment (atstoyanov) [inline src/pages/User/UserProfilePage.tsx:32]: Not needed here if there is just one theme.

File: src/themes/default/defaults.ts [hunk 5]
```diff
@@ -7,8 +7,8 @@ import { typographyOptions } from './typography';
 export const theme: ThemeOptions = {
   palette: paletteOptions,
   typography: typographyOptions,
-  shape: { borderRadius: 5 },
-  spacing: 10,
+  shape: { borderRadius: 4 },
+  spacing: 8,
```
Comment (atstoyanov) [inline src/themes/default/defaults.ts:11]: These are by default.

**Review by atstoyanov — CHANGES_REQUESTED** [review]

**Comment by Enderedne** [comment]
We don't want the changes to go to the develop branch

**Review by atstoyanov — APPROVED** [review]

**Pull request merged**